src/styles/components/_header.scss

```diff
@@ -13,6 +13,18 @@
     > hr {
         opacity: 0.02;
     }
+
+    &-round-count {
+        display: flex;
+        gap: map-get($space-sizes, small);
+        align-items: center;
+
+        > input {
+            @extend h1;
+            max-width: 60px;
+            border: 1px solid rgba(map-get($colors, contrast), 0.25);
+        }
+    }
```

Comment:
Not a fan of the border. My want would for the text to look as if it was in the same element as the header element. Reducing the gap as well as getting rid of the border should mostly suffice. Also the text font-family doesnt match. inherit fixes this.
```suggestion
    &-round-count {
        display: flex;
        align-items: center;

        > input {
            @extend h1;
            max-width: 40px;
            font-family: inherit;
        }
    }
```

Also the raleway font is [a bit wierd](https://www.reddit.com/r/graphic_design/comments/4l64zo/i_enjoy_the_raleway_font_a_lot_but_damn_those/). Adding the following to the global "*" styles or input styles should fix the number wierdness (as noted in the linked webpage).

```css
# src/styles/_global.scss
* {
    box-sizing: border-box;
    margin: 0%;
    padding: 0%;
    min-width: 0%;
    font-variant-numeric: lining-nums;
    -moz-font-feature-settings:"lnum" 1; 
    -moz-font-feature-settings:"lnum=1"; 
    -ms-font-feature-settings:"lnum" 1; 
    -o-font-feature-settings:"lnum" 1; 
    -webkit-font-feature-settings:"lnum" 1; 
    font-feature-settings:"lnum" 1;
}
```